spec/classes/init_spec.rb

```diff
@@ -386,6 +386,43 @@
           it { is_expected.to contain_ini_setting('fallback_ntp_server') }
         end
 
+        context 'when setting timezone' do
+          let(:params) do
+            {
+              timezone: 'America/Chicago',
+            }
+          end
+
+          it { is_expected.to compile.with_all_deps }
+          it { is_expected.to contain_exec('set system timezone') }
```

Comment:
Update to make use of the param:

```suggestion
          it { is_expected.to contain_exec('set system timezone').with_command('timedatectl set-timezone America/Chicago') }
```